org-formation/070-guard-duty/_tasks.yaml

```diff
@@ -0,0 +1,75 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'guardduty'
+
+  accountId:
+    Type: String
+    Description: The identifier from the account used to manage GuardDuty
+    Default: !Ref SecurityCentralAccount
+
+GuardDuty:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/GuardDuty/guard-duty.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}'
+  StackDescription: GuardDuty - Base
+  TerminationProtection: true
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+  OrganizationBindings:
+    LogArchiveBinding:
+      Account: !Ref LogArchiveAccount
+    MemberBinding:
+      Account: '*'
+      IncludeMasterAccount: true
+      ExcludeAccount:
+        - !Ref accountId
+    AllBinding:
+      Account: '*'
+      IncludeMasterAccount: true
+  Parameters:
+    resourcePrefix: !Ref resourcePrefix
+    accountId: !Ref accountId
+
+# bucket that contains the trusted IP addresses that are excluded from GuardDuty findings
+GuardDutyTrustedIpsBucket:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/GuardDuty/trusted-ips-bucket.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-trusted-ips-bucket'
+  StackDescription: GuardDuty - Trusted IPs bucket
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+  TerminationProtection: true
+  Parameters:
+    resourcePrefix: !Ref resourcePrefix
+    bucketName: !Sub '${resourcePrefix}-${appName}-trusted-ips'
+
+GuardDutyTrustedIps:
```

Comment:
I think we should leave this out as its unnecessary

Reply:
it's optional.  it doesn't do anything if you don't put any values into trusted-ips.txt file.